pkg/controller/admissionchecks/provisioning/controller.go

```diff
@@ -542,18 +541,24 @@ func (c *Controller) syncCheckStates(ctx context.Context, wl *kueue.Workload, ch
 					updated = updateCheckMessage(&checkState, apimeta.FindStatusCondition(pr.Status.Conditions, autoscaling.CapacityRevoked).Message) || updated
 				}
 			case isBookingExpired(pr):
-				if !workload.IsAdmitted(wl) {
-					attempt := getAttempt(log, pr, wl.Name, check)
-					if attempt <= c.maxRetries {
-						// it is going to be retried
-						message := fmt.Sprintf("Retrying after booking expired: %s", apimeta.FindStatusCondition(pr.Status.Conditions, autoscaling.BookingExpired).Message)
+				if workload.IsAdmitted(wl) {
```

Comment:
Maybe include as part of the prep PR in the other comment: https://github.com/kubernetes-sigs/kueue/pull/3375/files#r1822708150